server/planning/tests/utils_tests.py

```diff
@@ -2,36 +2,37 @@
 from datetime import datetime
 from planning.utils import get_event_formatted_dates
 from planning.search.queries import elastic
+import pytz
 
 
 class TestGetEventFormattedDates(TestCase):
     def test_multi_day_event(self):
-        start = datetime(2024, 5, 28, 5, 00, 00)
-        end = datetime(2024, 5, 29, 6, 00, 00)
-        event = {"dates": {"start": start, "end": end}}
+        start = datetime(2024, 5, 28, 5, 00, 00, tzinfo=pytz.utc)
```

Comment:
not sure if it would change the tests, but afaik we won't have tzinfo set there, so would do the tests without it

Reply:
we includes timezone info in the start and end datetime of an event:- 
<img width="666" alt="image" src="https://github.com/superdesk/superdesk-planning/assets/73937490/25829a1b-fdf4-4090-a2bb-01335959466b">

Reply:
yep it looks like it, but that's just how it's formatted it's always adding `+0000`. in db it's stored without timezone afaik

Reply:
[here](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/assignments/assignments.py#L460) also I checked the date time we got in this format `2024-04-27 18:30:00+00:00`, that's why I added `tzinfo`

Reply:
when we didn't include timezone information in the datetime objects for the test cases, the tests failed inconsistently on our CI system.